src/detect_breakends.cpp

```diff
@@ -91,6 +105,12 @@ int main(int argc, char ** argv)
         return -1;
     }
 
+    // default settings if no parameters given
+    if (args.methods.size() == 0)
+    {
+        args.methods = {cigar_string, split_read, read_pairs, read_depth};  // default is using all methods
+    }
+
```

Comment:
See above. I would set it as default in `args` unless it does some funny help page stuff.
If it does (and hence you want the default in `args` to be empty), the `add_option` should note that default = all methods.